designs/developer/4.6/HPA/index.md

```diff
@@ -0,0 +1,84 @@
+---
+parent: Developer
+version: 4.6
+---
+
+# Horizontal Pod Autoscaler
+
+## Description
+
+Horizontal pod autoscalers can be applied to workloads so that pods will scale up and down as needed so resources are not being wasting when waiting for peak resources. Horizontal pod autoscalers can only be applied to Deployment/Deployment Config workloads, not Knative services. A user can add, edit, and remove a horizontal pod autoscaler. 
+
+
+## Add a Horizontal Pod Autoscaler
+
+To add a horizontal pod autoscaler from a deployment or deployment config, the user clicks the Actions dropdown and selects 'Add Horizontal Pod Autoscaler'. 
+
+![Add horizontal pod autoscaler](img/1-hpa.jpg)
+
+### Add Horizontal Pod Autoscaler- Form
+
+The user can input Horizontal Pod Autoscaler attributes via a form or YAML editor. The form features limited field options so if the user wants to add custom metrics they should use the YAML editor. The info alert featured at the top of the form makes the user aware of this.
+
+![Add horizontal pod autoscaler form](img/2-hpa.jpg)
+
+In order to add a horizontal pod autoscaler to a workload the user has to set CPU and memory resource limits beforehand. If the user has not done that yet, an alert appears above the form view letting them know that they have to set CPU and memory resource limits before they can set values for the CPU and memory utilization fields. Once the user has set those resource limits the user will be able to set values for CPU and memory utilization.
+
+![Add horizontal pod autoscaler form alert](img/2.1-hpa.jpg)
+
+### Add Horizontal Pod Autoscaler- YAML Editor
+
+The user can toggle between the two pages view, either a form or YAML view. In the YAML view, the user can add custom metrics and inputs to the YAML editor. The schema for the horizontal pod autoscaler is featured on the right hand side.
+
+![Add horizontal pod autoscaler YAML editor](img/3-hpa.jpg)
+
+If the user has not set CPU or memory resource limits the same alert featured on the form view is displayed in the YAML view.  
```

Comment:
same comment from about the alert. Not sure if it needs to be slightly different wording for the YAML vs the form